src/common/components/PageHeader/PageHeader.tsx

```diff
@@ -10,6 +10,7 @@ export type PageHeaderSortByOptions = {
 }
 
 export type PageHeader = {
+  showTitle?: boolean
```

Comment:
You can change `title` prop to be optional and then no need for extra `showTitle` prop